staging/src/k8s.io/component-base/version/version.go

```diff
@@ -134,15 +139,15 @@ func (m *effectiveVersion) Validate() []error {
 	emulationVersion := m.emulationVersion.Load()
 	minCompatibilityVersion := m.minCompatibilityVersion.Load()
 
-	// emulationVersion can only be 1.{binaryMinor-1}...1.{binaryMinor}.
+	// emulationVersion can only be 1.{binaryMinor-3}...1.{binaryMinor}
 	maxEmuVer := binaryVersion
-	minEmuVer := binaryVersion.SubtractMinor(1)
+	minEmuVer := binaryVersion.SubtractMinor(3)
 	if emulationVersion.GreaterThan(maxEmuVer) || emulationVersion.LessThan(minEmuVer) {
 		errs = append(errs, fmt.Errorf("emulation version %s is not between [%s, %s]", emulationVersion.String(), minEmuVer.String(), maxEmuVer.String()))
 	}
-	// minCompatibilityVersion can only be 1.{binaryMinor-1} for alpha.
-	maxCompVer := binaryVersion.SubtractMinor(1)
-	minCompVer := binaryVersion.SubtractMinor(1)
+	// minCompatibilityVersion can only be 1.{binaryMinor-3} to 1.{binaryMinor}
+	maxCompVer := emulationVersion
```

Comment:
this is letting the compatibility version be == binary version? that would mean we'd allow a cluster to start on 1.33 and not be compatible with a 1.32 apiserver... are we really intending to relax that way in this PR?

Reply:
minCompVer is not exposed with any flag yet, and it stays at the default emu-1. Eventually we want to support compatibility version be == binary version. This is here so that we do not need to change it later.

Reply:
This was a bug. Per the [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/4330-compatibility-versions/README.md#motivation)

> Lastly, a --min-compatibility-version can be set to the binary version for new clusters being used for green field projects, providing immediate access to the latest Kubernetes features without the need to wait an additional release for features to settle in as is typically needed for rollback support.

Default is still at emulationVersion - 1.

Reply:
I know this was the aspirational end state for the KEP, but I didn't know if we were ready to relax to that yet. If it's not wired to a flag yet so there's no way to change it from emulationVersion-1 at this point, that's fine.